DFE-Digital/early-careers-framework
# New DQT API authentication and URLs

## Context

The TRA have released a new version of their DQT API for Qualified Teachers, this has a new authentication system using just a BEARER TOKEN and a new domain name.

## Tech review

### Is there anything that the code reviewer should know?

### Code quality checks
- [ ] Only removed redundant automated tests

### HTML Checks
- [ ] No pages have changed

## Product review

### How can someone see it it review app?
1. Click the link to review app (posted by a `github-actions` bot below)
2. Check responses from DQT to ensure environments are linked appropriately

#### Example teachers

##### DQT Test

Name | TRN | DoB | NINo
-- | -- | -- | --
Mostly Null | 1001001 | 1996-07-02 |  
Nested Null | 1001002 | 1996-07-02 |  
Mostly Populated | 1001000 | 1996-07-02 | AB123456D

##### DQT PreProd

Name | TRN | DoB | NINo
-- | -- | -- | --
Mostly Null | 2908308 | 1996-07-02 |  
Nested Null | 2908309 | 1996-07-02 |  
Mostly Populated | 2908310 | 1996-07-02 | AB123456D
